examples/contracts-and-usage-policy/templates/EventRestrictedUsage/EVENT_AGREEMENT_TEMPLATE.jsonld

```diff
@@ -13,18 +13,17 @@
         "ids:assignee": (idsc:PARTICIPANT_TEMPLATE),              // same value as ids:consumer,
         "ids:target": (idsc:ASSET_TEMPLATE),
         "ids:action": (idsc:ACTION_TEMPLATE),
-        "ids:constraint": {
+        "ids:constraint": [{
             "@type": "ids:Constraint",
             "ids:leftOperand": {
-              "@id": "?EventURI",                                 // The identifier of this event, for instance http://dbpedia.org/resource/Category:FIFA_World_Cup_tournaments
-              "ids:broader" : "idsc:EVENT"
+              "@id": "idsc:EVENT"
             },
             "ids:operator": { "@id": "idsc:EQUALS" },             // if pipReturnValue == true --> Constraint is staisfied
             "ids:rightOperand": {
-                "@value": "true",
-                "@type": "http://www.w3.org/2001/XMLSchema#boolean"
+                "@value": "?eventUri",
+                "@type": "xsd:anyUri"
```

Comment:
Please note that the "URI" part of "xsd:anyURI" needs to be in capital letters entirely